AmericanMedicalAssociation/AMA-style-guide
# Ama 24 hero cta image atom

https://palantir.atlassian.net/browse/AMA-24

Adds the image atom for Hero CTA, and positions the image and text side-by-side. Commits also update default Patternlab classes 'l', 'li', and 'cl' to layout, layout-item, and clearfix respectively.

Steps to test:

1. Navigate to Atoms > Images > Landscape-4x2.5
2. Observe that Landscape-4x2.5 is an atom.
3. Navigate to Molecules > Blocks > Hero CTA.
4. Observe that the Hero CTA molecule displays the Landscape-4x2.5 atom, and that it is positioned next to the text.
